scripts/src/main/resources/scripts/functions

```diff
@@ -701,8 +701,17 @@ function doExtract() {
     fi
   elif doIsInstallerExtension "${filename}"
   then
-     doRunCommand "mv \"${1}\" \"${target_dir}\""
+    doRunCommand "mv \"${1}\" \"${target_dir}\""
     extracted="no"
+  elif [ "${ext}" = ".msi" ]
+  then
+    local windows_download_file
+    windows_download_file=$(cygpath -w "${1}")
+    local windows_target_dir
+    windows_target_dir=$(cygpath -w "${target_dir}")
+    echo "Start-Process powershell.exe -ArgumentList \"msiexec /a ${windows_download_file} /qn TARGETDIR=${windows_target_dir}\" -Wait -NoNewWindow" >"${TMP}"/test.ps1
+    < "${TMP}"/test.ps1 powershell.exe
+    [ -f "${TMP}"/test.ps1 ] && rm "${TMP}"/test.ps1
```

Comment:
Instead of creating a script to run in powershell another powershell that runs our actual command and then deleting the script again, we can simply call the command directly.
The only special case to consider is that we are in bash here and have to use double slashes for windows command options:
```
msiexec //a "${windows_download_file}" //qn "TARGETDIR=${windows_target_dir}"
```

Reply:
A great solution, I will implement this.